decidim-core/app/presenters/decidim/resource_presenter.rb

```diff
@@ -31,15 +31,15 @@ def handle_locales(content, all_locales, &block)
 
     # Prepares the HTML content for the editors with the correct tags included
     # to identify the mentions.
-    def editor_locales(data, all_locales, extras: true)
+    def editor_locales(data, all_locales)
       handle_locales(data, all_locales) do |content|
         [
           Decidim::ContentRenderers::BlobRenderer,
           Decidim::ContentRenderers::UserRenderer,
           Decidim::ContentRenderers::MentionResourceRenderer
         ].each do |renderer_class|
           renderer = renderer_class.new(content)
-          content = renderer.render(links: false, editor: true, extras:).html_safe
+          content = renderer.render(links: false, editor: true).html_safe
```

Comment:
```suggestion
          content = renderer.render(editor: true).html_safe
```

The `links: false` object can't be removed?